demos/form/form.php

```diff
@@ -56,13 +57,16 @@
 $group->addControl('surname');
 $group->addControl('gender', [Form\Control\Dropdown::class, 'values' => ['Female', 'Male']]);
 
+$accordionLayout = $form->layout->addSubLayout([\Atk4\Ui\Form\Layout\Section\Accordion::class]);
+$details = $accordionLayout->addSection('Validating Field in Accordion');
```

Comment:
The phpstan error is correct here and I am still not fully decided id we want https://github.com/atk4/ui/pull/1679 or not, thus add this to `phpstan.neon.dist` at the "most logical place" - around the same phpstan errors/level and file names are sorted alphabetically